docs/beginner-checks.md

```diff
@@ -0,0 +1,77 @@
+# Getting Started with Scorecard Checks for Supply Chain Security
+
+If you are an open-source developer, you know that maintaining the security posture of a project can feel like a full-time job, but it can be easier with a little guidance. [Scorecard](https://github.com/ossf/scorecard#openssf-scorecard), an automated tool created [by the Open Source Security Foundation](https://securityscorecards.dev/) (OpenSSF), checks projects for security best practices and scores projects on a scale between 1 and 10. Maintainers can use the Scorecard checks as guidelines to improve their project's security practices.
```

Comment:
Should the OpenSSF link go to the securityscorecards page? or https://openssf.org/ ?

Reply:
Thanks, that makes sense, I changed the link.